src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  apiUrl: 'http://dev.algorea.org/api'
```

Comment:
Should even be api as it runs on the same domain. (will handle http/https better)
```suggestion
  apiUrl: 'api'
```

Reply:
Okay

Reply:
You could have just commit the proposal just above ("commit suggestion")